rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -234,18 +265,112 @@ For example, if you work in Buenos Aires (ART (UTC -3)), specify a schedule that
+|`5 4 3 2 1` | At 04:05 AM on every Monday in every February and (?) on the 3rd of every February| This Cron expression is invalid. You can set either the day of the week or day of the month, but not both. Replace either the day of the week or day of the month with an asterisk. If you want to run a process exactly once on a specific time, you must specify the day of the month but not the day of the week. See next two examples. 
+|`5 4 3 2 *` | At 04:05 AM on the 3rd of every February| If you want to run a process exactly once on a specific time, you must specify the day of the month but not the day of the week. You can't specify the year. 
+|`5 4 * 2 3` | At 04:05 AM on every Wednesday of every February| If you want to run a process exactly once on a specific time, you must specify the day of the month but not the day of the week.You can't specify the year. 
+|`*/7 * * * *` |Run at minute 00, 07, 14, 21, 28, 35, 42, 49, 56 of every hour (of every day of every month on every weekday). |Starts at minute 0 because it's the first of all values in the valid range (*).
+|`45/05 * * * *` |Run at minutes 45, 50, and 55 of every hour |
+|`* * * * *` |Run every minute forever |
+|`* 9-16 * * 1-5` |Every minute, between 09:00 AM and 04:59 PM, Monday through Friday |
+|`00,10,42 9-16 1-10,20-30 2-7,12 *`` |At 0, 10, and 42 minutes past the hour, between 09:00 AM and 04:59 PM, on day 1 through 10 and 20 through 30 of the month, and on Friday, only in February through July and December |You can combine lists and ranges.
+|===
+
+Non-existant times, such as 'missing hours' during daylight savings conversion, never match the bot's time-date setting, causing processes scheduled during the 'missing times' not to run. Similarly, times that occur more than once (again, during daylight savings conversion) cause matching processes to run twice.
+
+'''
+=====
++
+** *Calendar Scheduling*
 +
+Use this option to edit run configurations that already use calendar scheduling.
++ 
+Add one or more fixed schedules within which the productive run is carried out continuously. For each schedule, define time intervals within which the process is executed. RPA Manager combines all schedules to compute the execution times.
++
+.How does calendar scheduling work?
+[%collapsible]
+=====
+
+'''
+
+You can create multiple schedules. These may also overlap. All schedules together make up a time frame within which the process may be executed. The executing bot checks every minute whether it's within this time frame or not. A new process run isn't started until all conditions have been fulfilled.
+
+For this reason, the executing bot checks every minute whether it's within a service time and whether the required process applications are outside all downtimes. The process isn't executed if the executing bot is outside its service time or if one or more of the process applications used by the bot are within a downtime.
+
+Running processes are always executed completely.
```

Comment:
What does this mean? That processes always run until they finish? Isn't this the normal/expected behavior? Why clarify? 

Also, what happens if the bot crashes or the machine hosting the bot crashes? In that case, isn't this statement false?

Reply:
The process might take longer to run than the pause between the Bot's Service Time takes. So it might indeed run outside service time.
Maintaining a bot might therefore lead to breaking a running process.
Crashs always interrupt any run. That should be clear to the user from software experience.

Reply:
Rephrased the paragraph